crypto/pipe.go

```diff
@@ -67,21 +64,25 @@ type decoder struct {
 }
 
 func NewDecoder(reader io.Reader, iv []byte, symKey []byte) *decoder {
-	decipher, _ := NewDeCipher(iv, symKey)
-	return &decoder{reader: reader, DeCipher: decipher}
+	decipher := NewDeCipher(iv, symKey)
+	return &decoder{reader: reader, DeCipher: *decipher}
 }
 
-func (c *decoder) DycryptOnFly(filePath string) error {
-	size:=16
+func (c *decoder) DecryptOnFly(filePath string) error {
```

Comment:
Have we changed this function's name at all places it wass used?

Reply:
I am not sure about this. If this code is used by other libraries, then it may break their functionality.

Reply:
got it. I just checked and it seems it is not used anywhere else